.changeset/rare-garlics-kneel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@web5/api": minor
```

Comment:
shouldn't this be a patch? is it a breaking change? I believe with minor versions we have to get devrel involved

Reply:
Taking a dependency on the latest version of `dwn-sdk-js` is a breaking change and we do want to get dev-rel involved.